#! /bin/sh -e
## 4_dict-pager.dpatch by Roland Mas <lolando@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: dict.el: call dict with --pager

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
diff -urNad 4.emacs-goodies-el.tmp/elisp/dict.el 4.emacs-goodies-el/elisp/dict.el
--- 4.emacs-goodies-el.tmp/elisp/emacs-goodies-el-extra/dict.el	2003-03-19 14:24:04.000000000 +0100
+++ 4.emacs-goodies-el/elisp/emacs-goodies-el-extra/dict.el	2003-03-19 14:26:03.000000000 +0100
@@ -321,6 +321,7 @@
    (if dict-raw " --raw" "")
    (if (not (= dict-pipesize 256)) (concat " --pipesize " (number-to-string dict-pipesize)) "")
    (if dict-client (concat " --client " dict-client) "")
+   " --pager -"
    " "))
 
 (defun dict-get-answer (word)
